src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+export const addBook = (book) => ({
+  type: 'ADDBOOK',
+  payload: book,
+});
+
+export const removeBook = (book) => ({
+  type: 'REMOVEBOOK',
+  payload: book,
+});
+
+const booksReducer = (books = [], action) => {
+  switch (action.type) {
+    case 'ADDBOOK':
+      return [...books, action.payload];
+    case 'REMOVEBOOK':
+      return books.filter((book) => book.id !== action.payload.id);
+    default:
+      return books;
+  }
+};
+
+export default booksReducer;
```

Comment:
- [ ] Nice work @V0pe , The value of each action type needs to follow the [ducks pattern], so it will be something like:       `const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`, kindly fix this

Reply:
@VICTOR-CHUKWU Thanks for clearing that out. I have rectified it.